packages/css/src/components/column/column.scss

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+@import "../../common/size";
+
+.amsterdam-column {
+  display: grid;
+}
+
+@each $size, $label in $amsterdam-sizes {
+  .amsterdam-column--#{$label} {
+    gap: var(--amsterdam-column-gap-#{size});
```

Comment:
This renders `gap: var(--amsterdam-column-gap-size);` currently, not `gap: var(--amsterdam-column-gap-md);` etc